src/components/link/bl-link.ts

```diff
@@ -0,0 +1,166 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { classMap } from "lit/directives/class-map.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import style from "./bl-link.css";
+
+export type LinkVariant = "inline" | "standalone";
+export type LinkSize = "large" | "medium" | "small";
+export type LinkKind = "primary" | "neutral";
+
+/**
+ * @tag bl-link
+ * @summary Baklava Link component for navigation
+ *
+ * @cssproperty [--bl-link-color=--bl-color-primary] Sets the color of link
+ * @cssproperty [--bl-link-hover-color=--bl-color-primary-hover] Sets the hover color of link
+ * @cssproperty [--bl-link-active-color=--bl-color-primary-active] Sets the active color of link
+ */
+
+@customElement("bl-link")
+export default class BlLink extends LitElement {
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+
+  /**
+   * Target URL for the link
+   */
+  @property({ type: String, reflect: true })
+  target = "";
```

Comment:
add type safety, we can inherit from anchor's types

Reply:
don't add them by hand as much as possible, there are examples of inheriting existing types from elements

Reply:
The target here is not the anchor type but the href. I want to give it a more meaningful name